docs/source/getting-started/installation.rst

```diff
@@ -11,7 +11,7 @@ The Daml Connect SDK currently runs on Windows, macOS and Linux.
 
 You need to install:
 
-1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
+1. `Visual Studio Code <https://code.visualstudio.com/download>`_. For best experience, make sure the ``code`` command is available (it is enabled by default on both Linux and Windows, though in both cases only for new terminals opened _after_ the installation; for macOS, follow additional instructions `here <https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line>`_).
```

Comment:
Not trying to be nit-picky really, but what's `best experience` within this context? Is this a must or nice_to_have? What's getting improved here?

Reply:
`daml studio` deliberately does not require `code` being in PATH and that’s what we tell people to use. So I’d just omit this.

Reply:
I'm a bit confused as to why Khuram thought he had to set that up then. 🤔

I'll update the wording to mention it's not needed.

Reply:
Or would you prefer I remove it completely?